packages/fxa-admin-server/src/subscriptions/appstore.service.spec.ts

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Test, TestingModule } from '@nestjs/testing';
+import { MockConfig, MockLogService } from '../mocks';
+import { AppStoreService } from './appstore.service';
+import { FirestoreFactory } from './firestore.service';
+
+describe('App Store Service', () => {
+  let service: AppStoreService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [
+        AppStoreService,
+        MockConfig,
+        FirestoreFactory,
+        MockLogService,
+      ],
+    }).compile();
+
+    service = module.get<AppStoreService>(AppStoreService);
+  });
+
+  it('should be defined', () => {
+    expect(service).toBeDefined();
```

Comment:
I can add a mock that ensures purchaseManager.queryCurrentSubscriptionPurchases is invoked properly. 

Really the test coverage should be on the purchase manager. IMO, I shouldn’t have re-test that the purchase manager is working here. Since the purchase manager and iap stuff was already in place, I sort of assumed it was already under test, but maybe it could use more test coverage. Having to add this coverage seems like a separate task, and I’d have to evaluate what’s already there.